wiki/History_of_osu!/2018/en.md

```diff
@@ -4,22 +4,58 @@
 
 ## January
 
-After a six-month long hiatus, flyte posted on the [osu!next](http://osunext.tumblr.com) blog with some teaser pictures for the next website design. He explained that the current state of the website had many UX problems, from various functionality specs to user feedback, that wasn't considered at the time when designing. Since then, he had been taking some time to redesign the website's redesign from scratch. ([osu!next](http://osunext.tumblr.com/post/169336245023/hi-its-been-6-months-since-my-last-post-rest))
+After a six-month long hiatus, flyte posted on the [osu!next](http://osunext.tumblr.com) blog with some teaser pictures for the next website design. He explained that the current state of the website had many UX problems, from various functionality specs to user feedback, that wasn't considered at the time when designing. Since then, he had been taking some time to redesign the website's redesign from scratch.
 
-The osu!mania 7K World Cup 2018 registrations opened. ([osu!mania 7K World Cup 2018: Registrations now open!](https://osu.ppy.sh/home/news/2018-01-10-MWC7K-2018-registrations-open))
+The osu!mania 7K World Cup 2018 registrations opened. Community Choice 2017 (previously known as the "Best of" series) voting was opened.
 
-osu!lazer's osu!catch game mode was vastly improved. ([Dean Herbert on Twitter](https://twitter.com/ppy/status/951884503122784257))
+In osu!lazer, background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hit sounds were added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
 
-Community Choice 2017 (previously known as the "Best of" series) voting was opened. ([Community Choice 2017](https://osu.ppy.sh/home/news/2018-01-18-community-choice-2017))
+In osu!web, pagination improvements for beatmap pack listings, hype and nomination counters were visible on the beatmap listings. Obtaining a permalink to beatmap discussion post was now easier to copy, localisations now resize the menu upon hovering over the locales star ratings can be seen in the beatmap listing via hovering the difficulty icon, and sorting of beatmap discussions were added. A limit to only load 250 forum threads and unload others was implemented to avoid heavy memory consumption.
```

Comment:
In osu!web, beatmap pack listings was improved, hype and nomination counters were added to the beatmap listings. Obtaining a permalink to beatmap discussion post was made easier to copy, localisations can now resize the menu upon hovering over the locales, star ratings can be seen on the beatmap listing via hovering the difficulty icon, and sorting of beatmap discussions was added. A limit to only load 250 forum threads and unload others was implemented to avoid heavy memory consumption.